ios/Classes/PTFlutterViewController.h

```diff
@@ -14,6 +14,15 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic) BOOL needsRemoteDocumentLoaded;
 @property (nonatomic) BOOL documentLoaded;
 
+// viewer options
+@property (nonatomic, copy, setter=setFitMode:) NSString* fitMode;
+@property (nonatomic, copy, setter=setLayoutMode: ) NSString* layoutMode;
+@property (nonatomic, assign, setter=setInitialPageNumber: ) int initialPageNumber;
+@property (nonatomic, assign, setter=setIsBase64: ) BOOL isBase64;
```

Comment:
Why are the settings being specified here? They appear to be using the default names.